keras/engine/training.py

```diff
@@ -2160,7 +2160,8 @@ def predict(
                 run until the input dataset is exhausted.
             callbacks: List of `keras.callbacks.Callback` instances.
                 List of callbacks to apply during prediction.
-                See [callbacks](/api_docs/python/tf/keras/callbacks).
+                See [callbacks](https://www.tensorflow.org/api_docs/python/tf/
+                keras/callbacks).
```

Comment:
Please make sure the link fits on a single line

Reply:
Made the changes